Detectors/PHOS/base/src/Hit.cxx

```diff
@@ -10,38 +10,44 @@
 
 #include "PHOSBase/Hit.h"
 
-ClassImp(o2::PHOS::Hit)
+ClassImp(o2::phos::Hit)
 
-using namespace o2::PHOS;
+  using namespace o2::phos;
```

Comment:
this is a bug in clang-format which can be fixed putting a `;` after the ClassImp or putting ClassImp at the end of the file